config/application.rb

```diff
@@ -16,6 +16,11 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
+    # Generates a "structure.sql" and allow raw sql
     config.active_record.schema_format = :sql
+    # ActiveJob abstraction
+    config.active_job.queue_adapter = :sidekiq
```

Comment:
maybe this can go just in production env file? For development I'm ok using sidekiq but we could choose to not use it. And for testing we should use `config.active_job.queue_adapter = :test`